_includes/logo.html

```diff
@@ -1,3 +1,8 @@
 <div class="usa-logo">
-  <em class="usa-logo__text"><a href="{{site.baseurl}}/" title="Home" aria-label="Home">{% asset searchgovlogo.png class="logo" alt="search.gov" width="175" height="30" %}</a></em>
+  <em class="usa-logo__text">
+    <a href="{{site.baseurl}}/" title="Home" aria-label="Home">
```

Comment:
If this link isn't working, we might need to add space characters inside of the curly brackets

Reply:
@dawnpm It seems like the link is working in the preview link - starting on a different [page](https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/afarooque/2020-redesign-netlifycms/about/customers.html) the logo links to the main homepage. Any requested changes here?

Reply:
ok, not needed then.